battlepage.html

```diff
@@ -21,22 +21,22 @@
     <div class="main container">
         <section id="battleview" class="row content">
             <div id="enemy" class="enemy col-sm-6 col-sm-push-6">
-                <header class="blood-flow">
-                    <div id="enemy-hp" class="jquery-ui-like-enemy enemyHP">
-                        <div>&nbsp;</div>
+                <div class="hp">
```

Comment:
same here, adding new class, and make sure non of the js code is doing something to hp class kind of domObj